templates/csharp/src/Appwrite/Services/Service.cs.twig

```diff
@@ -0,0 +1,12 @@
+namespace Appwrite
```

Comment:
We can use a twig var here, instead of Appwrite hard-coded, might use it for other stuff in the future